frontend/src/containers/MetricsStep/index.tsx

```diff
@@ -68,24 +69,49 @@ const MetricsStep = () => {
   const shouldGetBoardConfig = useAppSelector(selectShouldGetBoardConfig);
 
   const getInfo = useCallback(
-    () =>
+    async () => {
       getBoardInfo({
         ...boardConfig,
-        startTime: dayjs(startDate).valueOf().toString(),
-        endTime: dayjs(endDate).valueOf().toString(),
+        dateRanges,
       }).then((res) => {
-        if (res.data) {
-          dispatch(updateBoardVerifyState(true));
-          dispatch(updateJiraVerifyResponse(res.data));
-          dispatch(updateMetricsState(merge(res.data, { isProjectCreated: isProjectCreated })));
-          dispatch(updateShouldGetBoardConfig(false));
-          dispatch(updateFirstTimeRoadMetricsBoardData(false));
+        if (res) {
+          if (res[0].data) {
+            const boardInfo = res?.map((r) => r.data);
+            const commonPayload = combineBoardInfo(boardInfo!);
+            dispatch(updateBoardVerifyState(true));
+            dispatch(updateJiraVerifyResponse(commonPayload));
+            dispatch(updateMetricsState(merge(commonPayload, { isProjectCreated: isProjectCreated })));
+            dispatch(updateShouldGetBoardConfig(false));
+            dispatch(updateFirstTimeRoadMetricsBoardData(false));
+          }
         }
-      }),
+      });
+    },
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [],
   );
 
+  const combineBoardInfo = (results: BoardInfoResponse[]) => {
```

Comment:
results: BoardInfoResponse[] -> `boardInfoResponseList: BoardInfoResponse[]`

results: what does it means,you know